index.html

```diff
@@ -3,44 +3,148 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>Felony Disenfranchisement</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link href='https://fonts.googleapis.com/css?family=Fjalla+One|Exo:400,600|Ropa+Sans' rel='stylesheet' type='text/css'>
   <style>
     .header {
-      text-align: center;
+      text-align: left;
     }
+    table {
+    border-collapse: collapse;
+    width: 100px;
+    }
+
+    th, td {
+    padding: 8px;
+    text-align: left;
+    border-bottom: 1px solid #ddd;
+    }
+
+    tr:hover td{font-weight: bold;
+    }
+
+    body {
+      background-color: #4C4C4C;
+          }
+
+    th {
+       font-family: 'Exo', sans-serif;
+        color: #FFFFFF;
+        width: 400px;
+                } 
+   h1{
+       font-family: 'Exo', sans-serif;
+        color: #FFFFFF;
+                }   
+    p,td {
+       font-family: 'Exo', sans-serif;
+        color: #B7B7B7
+                }
   </style>
 </head>
 <body>
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <h1>Felony Disenfranchisement</h1>
+      <p>States with the 15 Highest Rates of Felony Disenfranchisement</p>
     </div>
     <table class="table">
       <thead>
         <tr>
-          <th>Fruit</th>
-          <th>Price</th>
-          <th>Quantity</th>
+          <th>State</th>
+          <th>Abbreviation</th>
+          <th>Type of Restrictions</th>
+          <th>Percentage of Felons Disenfranchised</th>
         </tr>
       </thead>
       <tbody>
         <tr>
-          <td>Apples</td>
-          <td>$0.55</td>
-          <td>5</td>
+          <td style="color: #FFFFFF;">Florida</td>
+          <td>FL</td>
+          <td>Restrictions Post-Sentence</td>
+          <td style="background-color: #BD0026;color: #FFFFFF;">10.42%</td>
```

Comment:
I can see your new changes on the same pull request. Alex and I will close each pull request once we have completed the grading.

Note that there's no obligation to respond to comments before class (although you're welcome to keep working after submitting a pull request). We'll continue to iterate on this project for next week's assignment, so these comments are intended to help guide that iteration.